### ISSUE TITLE
A Different Kind of Request

### ISSUE THREAD

**Issue by GoobertFroobert** [babybluetit/Xaeros-Minimap-Modded-Support#89]
Is it at all possible to support other _resourcepacks_, and if so, would you be willing to do so? And if not, would you be willing to make a tutorial on how to do it ourselves if it _is_ possible?

I am looking to add support for [Fresh Animations](https://www.curseforge.com/minecraft/texture-packs/fresh-animations), as it is the main resourcepack I play with. However, it disables the radar Icon functionality when enabled. 

Thank you for your time, and I hope to hear back from you soon.

**Comment by thexaero** [comment]
I know you're not asking me, but yes, it is possible. Would have to be a separate resource pack from this though.

**Comment by CLS-Ghost350** [comment]
Could you add support for vanilla mobs? Having other resourcepacks enabled can break the default mob icons from rendering.

**Comment by GoobertFroobert** [comment]
> Could you add support for vanilla mobs? Having other resourcepacks enabled can break the default mob icons from rendering.

Yeah, that would also be extremely helpful, I hope to see a feature like that get added sometime soon.
